README.md

```diff
@@ -1 +1,51 @@
-# Review_Project_Trial
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review_Project_Trial
+
+> One paragraph statement about the project.
```

Comment:
It is better to remove placeholder text such as this out of the README. Placeholder text makes your README look unprofessional. You can add the actual statement describing the project here.